dockernet/tests/integration_tests.bats

```diff
@@ -199,6 +193,56 @@ setup_file() {
   assert_equal "$stake_diff" "1"
 }
 
+@test "[INTEGRATION-BASIC-$CHAIN_NAME] LSM liquid stake" {
+  if [[ "$CHAIN_NAME" != "LSM" ]]; then
+    skip "Skipping LSM liquid stake for chains without LSM support" 
+  fi
+
+  staker_address_on_host=$($HOST_MAIN_CMD keys show $USER_ACCT -a)
+  staker_address_on_stride=$($STRIDE_MAIN_CMD keys show $USER_ACCT -a)
+  validator_address=$(GET_VAL_ADDR $CHAIN_NAME 1)
+
+  # delegate on the host chain
+  $HOST_MAIN_CMD tx staking delegate $validator_address ${TRANSFER_AMOUNT}${HOST_DENOM} --from $USER_ACCT -y
+  WAIT_FOR_BLOCK $STRIDE_LOGS 2
+
+  # tokenize shares
+  $HOST_MAIN_CMD tx staking tokenize-share $validator_address ${TRANSFER_AMOUNT}${HOST_DENOM} $staker_address_on_host --from $USER_ACCT -y --gas 1000000
+  WAIT_FOR_BLOCK $STRIDE_LOGS 2
+
+  # get the record id from the tokenized share record
+  record_id=$($HOST_MAIN_CMD q staking last-tokenize-share-record-id | awk '{print $2}' | tr -d '"')
+
+  # transfer LSM tokens to stride
+  $HOST_MAIN_CMD tx ibc-transfer transfer transfer $HOST_TRANSFER_CHANNEL \
+    $staker_address_on_stride ${TRANSFER_AMOUNT}${validator_address}/${record_id} --from $USER_ACCT -y
+  
+  WAIT_FOR_BLOCK $STRIDE_LOGS 8
+
+  lsm_token_ibc_denom=$(GET_IBC_DENOM $STRIDE_TRANSFER_CHANNEL ${validator_address}/${record_id})
+
+  # get initial balances
+  sttoken_balance_start=$($STRIDE_MAIN_CMD q bank balances $staker_address_on_stride --denom st$HOST_DENOM | GETBAL)
+  lsm_token_balance_start=$($STRIDE_MAIN_CMD q bank balances $staker_address_on_stride --denom $lsm_token_ibc_denom | GETBAL)
+
+  # LSM-liquid stake
+  $STRIDE_MAIN_CMD tx stakeibc lsm-liquid-stake $STAKE_AMOUNT $lsm_token_ibc_denom --from $USER_ACCT -y 
+  WAIT_FOR_BLOCK $STRIDE_LOGS 2
```

Comment:
Seems like a good place to use the WAIT_FOR_BALANCE_CHANGE() since that is what the next lines explicitly care about

Reply:
done